src/Producer/SyncProcess.php

```diff
@@ -27,13 +27,13 @@ public function __construct()
     }
 
     /**
-     * @param string[][] $data
+     * @param mixed[][] $messagesData
      *
      * @return mixed[]
      *
      * @throws \Kafka\Exception
      */
-    public function send(array $data): array
+    public function send(array $messagesData): array
```

Comment:
In Kafka terminology this is a record set, so that's the name we should be using =)